### PR TITLE
Feature/fetch vertices on multi tags

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Rest of https://github.com/vesoft-inc/nebula/pull/2222

### CODE REVIEW

File: src/parser/TraverseSentences.h [hunk 1]
```diff
@@ -406,14 +406,18 @@ class FetchEdgesSentence final : public Sentence {
         return yieldClause_.get();
     }
 
-    std::string* edge() const {
-        return edge_.get();
+    const std::string* edge() const {
```
Comment (laura-ding) [inline src/parser/TraverseSentences.h:409]: the edge also support multi labels, the `edge()` function is needed to modify `edges()`

**Review by laura-ding — COMMENTED** [review]
I can't see the `fetch multi edges` in the python test

File: src/parser/TraverseSentences.h (hunk 1 above)
Comment (Shylock-Hg) [reply src/parser/TraverseSentences.h:409]: Don't support in fact.

File: tests/query/v1/test_fetch_vertex.py [hunk 2]
```diff
@@ -335,6 +335,144 @@ def test_fetch_vertex_get_all_with_yield_invalid(self):
...
+        expect_result = [['Tim Duncan', 'Tim Duncan', 42, "Tim Duncan", "psychology"],
+                         ['Boris Diaw', 'Boris Diaw', 36, T_EMPTY, T_EMPTY]]
+        self.check_resp_succeeded(resp)
+        self.check_out_of_order_result(resp, expect_result)
+
+        # from input
+        query = '''GO FROM "Boris Diaw" over like YIELD like._dst as id
+            | FETCH PROP ON player, bachelor $-.id'''
+        resp = self.execute_query(query)
+        expect_column_names = ['_vid', 'player.name', 'player.age', 'bachelor.name', 'bachelor.speciality']
```
Comment (yixinglu) [inline tests/query/v1/test_fetch_vertex.py:360]: why the column name is different between line 360 and line 377, `_vid` vs `VertexID` ?

File: src/parser/Clauses.h [hunk 3]
```diff
@@ -379,5 +379,47 @@ using InBoundClause = BoundClause;
...
+    }
+
+    std::vector<const std::string*> labels() const {
+        std::vector<const std::string*> labels;
+        labels.reserve(labels_.size());
+        for (const auto& it : labels_) {
+            labels.emplace_back(it.get());
+        }
+        return labels;
+    }
```
Comment (yixinglu) [inline src/parser/Clauses.h:409]: Why not return labels_ directly? what's the purpose of this copying?

File: tests/query/v1/test_fetch_vertex.py (hunk 2 above)
Comment (Shylock-Hg) [reply tests/query/v1/test_fetch_vertex.py]: Good catch. It's a mistake. Had fixed.

File: src/parser/Clauses.h (hunk 3 above)
Comment (Shylock-Hg) [reply src/parser/Clauses.h:409]: Keep same with others like the YIeldClumns::columns.

Comment (yixinglu) [reply src/parser/Clauses.h:409]: Not need it. We could fix that in the future.

Comment (Shylock-Hg) [reply src/parser/Clauses.h:409]: Ok, better to fix them together in another PR.

**Review by yixinglu — APPROVED** [review]
LGTM

**Review by laura-ding — APPROVED** [review]

**Comment by dutor** [comment]
Please resolve the conflicts.

**Comment by Shylock-Hg** [comment]
> Please resolve the conflicts.

Fixed.

**Review by CPWstatic — APPROVED** [review]
LGTM.

**Review by yixinglu — APPROVED** [review]

**Pull request merged**